### PR TITLE
Added optional unified resource namespace

### PR DESCRIPTION
**Pull request by 8tomat8**

### CODE REVIEW

File: soap/client.go [hunk 1]
```diff
@@ -94,6 +95,7 @@ func doRoundTrip(c *Client, setHeaders func(*http.Request), in, out Message) err
 	setXMLType(reflect.ValueOf(in))
 	req := &Envelope{
 		EnvelopeAttr: c.Envelope,
+		URNAttr: 	  c.URNamespace,
```
Comment (tandr) [inline soap/client.go]: `go fmt` before merge would be a great way to keep code consistently formatted.

File: soap/client.go [hunk 2]
```diff
@@ -224,6 +226,7 @@ type Envelope struct {
 	XMLName      xml.Name `xml:"SOAP-ENV:Envelope"`
 	EnvelopeAttr string   `xml:"xmlns:SOAP-ENV,attr"`
 	NSAttr       string   `xml:"xmlns:ns,attr"`
+	URNAttr      string   `xml:"xmlns:urn,attr,omitempty"`
 	TNSAttr      string   `xml:"xmlns:tns,attr"`
```
Comment (tandr) [inline soap/client.go]: You can save the next fella a trouble (in !97) and add `omitempty` for `TNSAttr` as well

Comment (8tomat8) [reply soap/client.go]: I've left comment in #97 . It will not work like that =)

Comment (tandr) [reply soap/client.go]: We have an envelope coming from 3rd party server that have this attribute missing, and xml parser is chocking on that. Will your fix alleviate that?

**Comment by fiorix** [comment]
This needs rebase before it can be merged.

**Comment by 8tomat8** [comment]
@fiorix done

**Comment by fiorix** [comment]
Cool, thanks! 👍

**Pull request merged**